### ISSUE TITLE
Virtual scroll not completing with custom components

### ISSUE THREAD

**Issue by Ross-Rawlins** [ionic-team/ionic#17298]
# Bug Report

**Ionic version:**
[x] 4.x

**Current behavior:**
When implementing virtual scroll with any content in the containing tag the page renders on start up. As soon as the only thing inside the container is the custom component it stays on the init process, the elements are given the correct positions but the class virtual loading is still appended to the element. This only gets removed when I scroll on the page and then the content shows.

**Expected behavior:**
The class virtual-loading show should be removed

**Steps to reproduce:**
Implement virtual scroll with a custom component inside of it.

**Related code:**
This renders
```
 <ion-virtual-scroll #virtualScroll [items]="eventsList"   approxItemHeight="400px">
        <div *virtualItem="let item" class="event-item">
1
            <app-events-list-item [event]="item" ></app-events-list-item>
        </div>
    </ion-virtual-scroll>
```
This does not
```
 <ion-virtual-scroll #virtualScroll [items]="eventsList"   approxItemHeight="400px">
        <div *virtualItem="let item" class="event-item">
            <app-events-list-item [event]="item" ></app-events-list-item>
        </div>
    </ion-virtual-scroll>
```

**Other information:**

**Ionic info:** 
```
Ionic:

   ionic (Ionic CLI)             : 4.6.0 (/Users/rossrawlins/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0
   @angular-devkit/build-angular : 0.12.3
   @angular-devkit/schematics    : 7.1.4
   @angular/cli                  : 7.1.4
   @ionic/angular-toolkit        : 1.2.2

System:

   NodeJS : v8.9.4 (/usr/local/bin/node)
   npm    : 6.6.0
   OS     : macOS Mojave
```

**Comment by Ross-Rawlins** [comment]
After further investigation to seems to only happen when I am setting the height of the custom component.

**Comment by Ross-Rawlins** [comment]
This issue with the class seems to also happen when you change the list.

**Comment by brandyscarney** [comment]
Thanks for the issue! This should be fixed in the `ionic 4.2.0-dev.201903262032.a8a48a4` dev release. Could you please try this and let us know if it isn't fixed?
